### ISSUE TITLE
Develop Landing Page Based on Script

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/demo-web-prestige#8]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
Cory completed the content script. It's time to develop it into an actual site.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
Develop the wireframe into actual site.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
clean slate.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented epilogue partial layout here: 16e4dcf8f2fceff3dd92780e7510c05f1658bd1f

**Comment by hollowaykeanho** [comment]
overall implemented in c7e4497cd31164e7aa7f19bfb8138245d49cbf2b. Removed Teams, footer, and etc out first and communicate to client as promised.
